x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -368,6 +368,10 @@ export default function ({ getService }: FtrProviderContext) {
           await ml.anomaliesTable.assertTableRowsCount(10);
         });
 
+        it('attaches swim lane embeddable to a case', async () => {
+          await ml.anomalyExplorer.attachSwimLaneToCase('viewBy');
```

Comment:
The test currently checks that the user can click through the workflow successfully. IMO it would be good to also check that the case has actually been created and ideally, that it contains the swim lane attachment.

If we just want to check case existance, this could probably be done by using one of the methods from [the existing API test utils](https://github.com/elastic/kibana/blob/main/x-pack/test/cases_api_integration/common/lib/utils.ts) via the [case api service](https://github.com/elastic/kibana/blob/main/x-pack/test/functional/services/cases/api.ts#L10-L15)).
If we also want to check that the swim lane is attached correctly, we should probably instead open the case in the UI and assert the attachments.

Reply:
Hey @pheyos , thanks for the review. Added more assertions in the last commits to validate the Cases page content. Also, checking with the [flaky test runner](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/1086)